### PR TITLE
19 mainlivecard design

### PR DESCRIPTION
**Pull request by Paajt**
# Description:
I have created the live event cards with hardcoded information using HTML and added styling to them with SASS. 

Issue #19 

# Type of change:
- [x] Created card structure with HTML (Container, Image, Title, Description, Book-Button). 
- [x] Added styling with SASS.
- [x] Information is hardcoded for now but it can/will be replaced in the future (see Issue #26)

# How Has This Been Tested or should be tested?
**Check in either**

- Branch now with npm run dev
- Pages once it is merged

# Additional information (screenshot, video, code examples):

Skärmbild 2024-12-22 092408

### CODE REVIEW

File: src/styles/styles.scss
Comment (Paajt) [inline src/styles/styles.scss:1]: Why is it formating this different tho? 🤔

Comment (alfredbrannare) [reply src/styles/styles.scss:1]: No clue, maybe it is prettier doing some kind of magic? 🥸

**Review by alfredbrannare — APPROVED** [review]
I can't see any issues and everything LG2M!

Nice job 😎

**Review by jennymakki — APPROVED** [review]
LG2M!

**Pull request merged**